server/src/main/resources/db/changelog/20180820091431-revert-share-fields.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="20180820091431-1" author="vrjain">
+        <comment> revert-share-fields</comment>
+        <dropColumn tableName="cp_pool" columnName="quantity_shared"/>
+        <dropColumn tableName="cp_pool" columnName="created_by_share"/>
+        <dropColumn tableName="cp_pool" columnName="has_shared_ancestor"/>
+        <dropColumn tableName="cp_consumer" columnName="recipient_owner_key"/>
```

Comment:
Do we need to drop the `cp2_product_shares` table?

Reply:
done